aswf-vfx-base/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM centos/devtoolset-6-toolchain-centos7
```

Comment:
This toolset gives us the right version of GCC, but we primarily use clang in our CI because it's faster and uses less memory. I attempted to yum install clang, but it gives me a very old version (3.4.2) which we no longer support, so I'm experimenting with installing a newer version:

```
yum install -y centos-release-scl
yum install -y llvm-toolset-7
scl enable llvm-toolset-7 bash
```

Reply:
Until we can use devtoolset-7 clang is not part of it, so I'll add a version compiled from source using the stdc++ library from the devtoolset, we use clang-7 internally, which version would you want?

Reply:
Right, yeah that would be ideal, clang-7 built against the stdc++ from devtoolset 6.